src/SprykerEco/Zed/ComputopApi/Business/Converter/CaptureConverter.php

```diff
@@ -13,24 +13,24 @@
 class CaptureConverter extends AbstractConverter implements ConverterInterface
 {
     /**
-     * @param array $response
+     * @param array $decryptedResponse
      *
      * @return \Generated\Shared\Transfer\ComputopApiCaptureResponseTransfer
      */
-    protected function getResponseTransfer(array $response): ComputopApiCaptureResponseTransfer
+    protected function getResponseTransfer(array $decryptedResponse): ComputopApiCaptureResponseTransfer
     {
         $computopApiResponseTransfer = new ComputopApiCaptureResponseTransfer();
-        $computopApiResponseTransfer->fromArray($response, true);
+        $computopApiResponseTransfer->fromArray($decryptedResponse, true);
         $computopApiResponseTransfer->setHeader(
-            $this->computopApiService->extractResponseHeader($response, $this->config->getCaptureMethodName())
+            $this->computopApiService->extractResponseHeader($decryptedResponse, $this->config->getCaptureMethodName()),
         );
         //optional fields
-        $computopApiResponseTransfer->setAId($this->computopApiService->getResponseValue($response, ComputopApiConfig::A_ID));
-        $computopApiResponseTransfer->setTransactionId($this->computopApiService->getResponseValue($response, ComputopApiConfig::TRANSACTION_ID));
-        $computopApiResponseTransfer->setAmount($this->computopApiService->getResponseValue($response, ComputopApiConfig::AMOUNT));
-        $computopApiResponseTransfer->setCodeExt($this->computopApiService->getResponseValue($response, ComputopApiConfig::CODE_EXT));
-        $computopApiResponseTransfer->setErrorText($this->computopApiService->getResponseValue($response, ComputopApiConfig::ERROR_TEXT));
-        $computopApiResponseTransfer->setRefNr($this->computopApiService->getResponseValue($response, ComputopApiConfig::REF_NR));
+        $computopApiResponseTransfer->setAId($this->computopApiService->getResponseValue($decryptedResponse, ComputopApiConfig::A_ID));
```

Comment:
Can we use method chaining here? e.g.
```
$computopApiResponseTransfer
    ->setAId(
        $this->computopApiService->getResponseValue($decryptedResponse, ComputopApiConfig::A_ID),
     )
    ->setTransactionId(
        $this->computopApiService->getResponseValue($decryptedResponse, ComputopApiConfig::TRANSACTION_ID),
     )
     ...
     ```
     Please check also the similar places below.

Reply:
Fixed